.release-please-manifest.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+{"detectors/node/opentelemetry-resource-detector-alibaba-cloud":"0.26.2","detectors/node/opentelemetry-resource-detector-aws":"1.0.3","detectors/node/opentelemetry-resource-detector-gcp":"0.26.2","detectors/node/opentelemetry-resource-detector-github":"0.26.1","metapackages/auto-instrumentations-node":"0.27.4","metapackages/auto-instrumentations-web":"0.27.2","packages/opentelemetry-browser-extension-autoinjection":"0.27.3","packages/opentelemetry-host-metrics":"0.27.1","packages/opentelemetry-id-generator-aws-xray":"1.0.1","packages/opentelemetry-test-utils":"0.29.0","plugins/node/instrumentation-tedious":"0.1.0","plugins/node/opentelemetry-instrumentation-aws-lambda":"0.29.0","plugins/node/opentelemetry-instrumentation-aws-sdk":"0.5.0","plugins/node/opentelemetry-instrumentation-bunyan":"0.27.1","plugins/node/opentelemetry-instrumentation-cassandra":"0.27.1","plugins/node/opentelemetry-instrumentation-connect":"0.27.1","plugins/node/opentelemetry-instrumentation-dns":"0.27.1","plugins/node/opentelemetry-instrumentation-express":"0.28.0","plugins/node/opentelemetry-instrumentation-generic-pool":"0.27.2","plugins/node/opentelemetry-instrumentation-graphql":"0.27.3","plugins/node/opentelemetry-instrumentation-hapi":"0.27.1","plugins/node/opentelemetry-instrumentation-ioredis":"0.27.1","plugins/node/opentelemetry-instrumentation-knex":"0.27.1","plugins/node/opentelemetry-instrumentation-koa":"0.28.1","plugins/node/opentelemetry-instrumentation-memcached":"0.27.1","plugins/node/opentelemetry-instrumentation-mongodb":"0.29.0","plugins/node/opentelemetry-instrumentation-mysql":"0.27.1","plugins/node/opentelemetry-instrumentation-mysql2":"0.28.0","plugins/node/opentelemetry-instrumentation-nestjs-core":"0.28.3","plugins/node/opentelemetry-instrumentation-net":"0.27.1","plugins/node/opentelemetry-instrumentation-pg":"0.28.0","plugins/node/opentelemetry-instrumentation-pino":"0.28.1","plugins/node/opentelemetry-instrumentation-redis":"0.28.0","plugins/node/opentelemetry-instrumentation-restify":"0.27.2","plugins/node/opentelemetry-instrumentation-router":"0.27.1","plugins/node/opentelemetry-instrumentation-winston":"0.27.1","plugins/web/opentelemetry-instrumentation-document-load":"0.27.1","plugins/web/opentelemetry-instrumentation-user-interaction":"0.28.1","plugins/web/opentelemetry-plugin-react-load":"0.26.1","propagators/opentelemetry-propagator-aws-xray":"1.0.1","propagators/opentelemetry-propagator-grpc-census-binary":"0.25.1","propagators/opentelemetry-propagator-ot-trace":"0.25.1","plugins/node/opentelemetry-instrumentation-fastify":"0.25.0"}
```

Comment:
i've manually fixed the conflict that https://github.com/open-telemetry/opentelemetry-js-contrib/commit/9f3b22774cad0721f9cbb18dbb9628bbf2e76982 introduced, if someone review it please check that i didn't make any mistake but it seems ok to me